Classes/Controller/MetadataController.php

```diff
@@ -460,7 +460,7 @@ private function parseCollections(int $i, $value, array &$metadata) : void
         foreach ($value as $entry) {
             $collection = $this->collectionRepository->findOneByIndexName($entry);
             if ($collection) {
-                $metadata[$i]['collection'][$j] = $collection->getLabel() ? : '';
+                $metadata[$i]['collection'] = $collection->getLabel() ? : '';
```

Comment:
In this case only the last read collection will be displayed to the user. Document can be part of more than one collection and each of them should be displayed in metadata list.

Reply:
Without the adjustments only the first letter of the label is inserted, the rest is then the remaining part of the index identifier of the collection, this is probably not intended. It may be necessary to make a further correction if there are several collections, but the existing version is incorrect in any case.